mitodl/mit-open
# Shanbady/add record hash field for hightouch sync

### What are the relevant tickets?

Closes #714

### Description (What does it do?)
This PR adds a "record_hash" field that hightouch requires for sync


### How can this be tested?
Checkout this branch. do a docker-compose  restart all or docker-compose down/up and see that the external "program certificate" table has a record_hash field.

### Additional Context
This field was added to a table sitting in an external schema and is completely unmanaged. The PR contains several migration files because the sequence of migrations looks like so:
1. seet model too unmanaged
2. introduce new nullable non-unique record_hash field
3. populate existing models using uuid
4. make record_hash non-nullable and unique
5. unset the 'managed' flag on programcertificate model
